pages/multiplayerGames.js

```diff
@@ -3,13 +3,14 @@ import React, { Component } from 'react';
 import { Router } from '../src/routes';
 
 import Bloc from '../src/components/bloc';
-import ListGames from '../src/components/listGames';
 import Button from '../src/components/button';
+import ListGames from '../src/components/listGames';
+import Image from '../src/components/image';
 import Page from '../src/components/page';
 import Row from '../src/components/row';
 import Section from '../src/components/section';
 
-import { games, join } from '../src/services/multiplayerGameService';
+import { games, join, newGame } from '../src/services/multiplayerGameService';
```

Comment:
`newGame` is badly named, it's a factory taking some config (API url) to build a method actually starting a new game.

Reply:
So it's the case for all methods in this service.

Should I use gamesFactory, joinFactory, newGameFactory ?

Reply:
Mostly depends on the naming rules of your project.  But even if the word `factory` usually scares developpers, it describes perfectly what it is there.